Simplenote/Classes/SPNoteEditorViewController.m

```diff
@@ -147,7 +144,13 @@ - (void)setupNavigationController
 {
     // Note: Our navigationBar *may* be hidden, as per SPSearchController in the Notes List
     [self.navigationController setNavigationBarHidden:NO animated:YES];
-    [self.navigationController setToolbarHidden:!self.searching animated:YES];
+
+    if (UIDevice.isPad) {
+        [self.navigationController setToolbarHidden:!self.searching animated:YES];
+    } else {
+        [self.navigationController setToolbarHidden:NO animated:YES];
+    }
```

Comment:
Suggestion!

```
BOOL mustHideToolbar = UIDevice.isPad && !self.searching;
[self.navigationController setToolbarHidden:mustHideToolbar animated:YES];
```

Reply:
Thinking about this, and I realized that I had separated setting up the buttons and displaying the buttons into different functions.  
I made this change, but also updated the code for endSearching to reset what the buttons are and hide them using the separate functions instead of inserting hide/don't hide into the setup function